archivist/notebooks/Find Asset and Create Attachment.ipynb

```diff
@@ -0,0 +1,331 @@
+    "# SECRET = represents the client secret from an Application Registration\n",
+    "RKVST_URL = getenv(\"URL\")\n",
+    "APPREG_CLIENT = getenv(\"CLIENT\")\n",
+    "APPREG_SECRET = getenv(\"SECRET\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "671c824f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def upload_attachment(arch, path, name):\n",
+    "    \"\"\"\n",
+    "    Obtains attachments from \"test_files\" folder and creates json payload\n",
+    "    for upload\n",
+    "    \"\"\"\n",
+    "    with pkg_resources.open_binary(test_files, path) as fd:\n",
+    "        blob = arch.attachments.upload(fd)\n",
+    "        attachment = {\n",
+    "            \"arc_display_name\": name,\n",
+    "            \"arc_attachment_identity\": blob[\"identity\"],\n",
+    "            \"arc_hash_value\": blob[\"hash\"][\"value\"],\n",
+    "            \"arc_hash_alg\": blob[\"hash\"][\"alg\"],\n",
+    "        }\n",
+    "        return attachment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "3e2d7571",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_artist(arch, name, artistid):\n",
+    "    \"\"\"\n",
+    "    Finds existing Artist asset by name and unique id\n",
+    "    \"\"\"\n",
+    "    attrs = {\n",
+    "        \"arc_display_type\": \"Artists\",\n",
+    "        \"arc_display_name\": name,\n",
+    "        \"artistid\": artistid,\n",
+    "    }\n",
+    "\n",
+    "    return arch.assets.read_by_signature(attrs=attrs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "a33b9635",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_image(arch, asset):\n",
+    "    \"\"\"\n",
+    "    Creates an event that uploads a primary image for an Artist asset\n",
+    "    \"\"\"\n",
+    "    attachments = upload_attachment(\n",
+    "        arch, \"pexels-andrea-turner-707697.jpeg\", \"arc_primary_image\"\n",
+    "    )\n",
+    "    attachment_list = [attachments]\n",
+    "\n",
+    "    props = {\"operation\": \"Record\", \"behaviour\": \"RecordEvidence\"}\n",
+    "    attrs = {\n",
+    "        \"arc_description\": \"Attaching an image\",\n",
+    "        \"arc_display_type\": \"Primary image\",\n",
+    "    }\n",
+    "\n",
+    "    asset_attrs = {\"arc_attachments\": attachment_list}\n",
+    "\n",
+    "    return arch.events.create(\n",
+    "        asset[\"identity\"], props=props, attrs=attrs, asset_attrs=asset_attrs\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "fbad039b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connecting to RKVST\n",
+      "URL https://app.rkvst.io\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "Main function of asset and event creation.\n",
```

Comment:
Capitalise. But actually this comment is wrong isn't it? This one isn't creating an Asset, it's finding one.